Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_cans.yml

```diff
@@ -288,6 +288,31 @@
   - type: Item
     sprite: Objects/Consumable/Drinks/dr_gibb.rsi
 
+- type: entity
+  parent: DrinkCanBaseFull
+  id: DrinkNukieCan
+  name: robust nukie drink
+  description: An encouraging drink... very encouraging, it is filled with green liquid, after use it is recommended to consult a doctor.
```

Comment:
"…very encouraging, it is filled…"

I think it should be more of a "…very encouraging. It is filled…" in order to be more grammaticaly correct.

Reply:
> "…very encouraging, it is filled…"
> 
> It should be more of a "…very encouraging. It is filled…" in order to be more grammaticaly correct.

Fixed it. I apologize, I use a translator, because of my mediocre knowledge of English.